PWGDQ/Core/VarManager.h

```diff
@@ -1593,6 +1595,34 @@ void VarManager::FillTrackCollision(T const& track, C const& collision, float* v
   }
 }
 
+template <uint32_t fillMap, typename T, typename C, typename M>
```

Comment:
How do you set magnetic field to "Propagator::Instance()" here? Could you pass o2::base::Propagator in filter PP task to this function as an argument, please?